wordpress-mobile/WordPress-iOS
# Add script to update appreleastorestrings.po from text files

This PR introduce a new script that updates the AppReleaseStoreString.po file with content from common text files. This update is aimed to avoid errors with .po syntax and to make it easier to update release notes and other metadata strings.

The script read the content from the following files:
 - `WordPress/Resources/release_notes.txt` for the release notes
 - `Scripts/fastlane/appstoreres/metadata/source/subtitle.txt` for the app subtitle in the App Store
 -  `Scripts/fastlane/appstoreres/metadata/source/description.txt` for the app description in the App Store
 - `Scripts/fastlane/appstoreres/metadata/source/keywords.txt` for the keywords in the App Store
 - `Scripts/fastlane/appstoreres/metadata/source/promo_screenshot_*.txt` for the screenshots promo messages
and it updates the `AppReleaseStoreString.po` file that is loaded by the translation system.

This  is another step for upgrading release automation.

To Test:
1. `cd Scripts` from project root folder
2. run `bundle install` to update the plugin version
3. change the content of some of the above .txt files ;
4. run `bundle exec fastlane update_appstore_strings version:10.7` (or any other version you want)
5. check that the `AppStoreString.po` file is correctly updated.
